### ISSUE TITLE
Display the field machine_name along with the label on the display mode

### ISSUE THREAD

**Issue by herbdool** [backdrop/backdrop-issues#6200]
## Description of the need

I'm dealing with a few content types that have many fields and some of them have the same label. It would be helpful to also display the machine_name alongside the field label when on a display mode form.

Here's a real example:

![2023-08-18 17 03 44 hda-chrl-bd lndo site 2f241804bdb3](https://github.com/backdrop/backdrop-issues/assets/1071483/45c1f0ca-15a0-4393-8a46-94a8de66ee51)

Which "Date" is which?

## Proposed solution

Something like this:

![2023-08-18 17 04 20 hda-chrl-bd lndo site 8ebf881c3ece](https://github.com/backdrop/backdrop-issues/assets/1071483/a40b7390-dc1e-44c4-8910-c7ee05a11625)

**Comment by argiepiano** [comment]
I've struggled with this so many times. This would be a great addition!

**Comment by olafgrabienski** [comment]
I'm not against the request, but out of curiosity: For which reason do you use the same label for different fields? Isn't that also confusing for content editors?

**Comment by kiamlaluno** [comment]
I guess that with many fields, it is possible to add a new field without noticing the same label has been already used.  
I am favorable to this change because I think that the field name should be information given on that page, accessible without checking a different page.

**Comment by klonos** [comment]
This is a UX improvement task, so it can go in at any time - not necessarily a minor release feature.

I'm also supportive of this 👍🏼

**Comment by herbdool** [comment]
@olafgrabienski this was all created before I got involved so I don't know for sure. I'm guessing the client figured it would be too wordy to make it "Staff Date" instead of just Date. They can see from the context that it follows staff signature so the date relates to the preceding signature. There were other fields with the same problem.
